Gemfile

```diff
@@ -30,7 +30,7 @@ gem 'foreman'
 gem 'gchartrb', require: 'google_chart'
 gem 'gdata', require: false
 gem "googlecharts", require: "gchart"
-gem 'haml-rails'
+# gem 'haml-rails'
```

Comment:
Why was this change necessary?

Reply:
So, this gemfile generates haml view files in the scaffold, which I commented out while generating the scaffold. I have added it back now in the latest commit.